Exec/Production/ChallengeProblem/challenge.inp

```diff
@@ -48,7 +49,7 @@ amr.v                = 1       # verbosity in Amr.cpp
 amr.data_log         = datlog extremalog
 
 # REFINEMENT / REGRIDDING
-amr.max_level       = 5       # maximum level number allowed
+amr.max_level       = 3       # maximum level number allowed
```

Comment:
I think technically the challenge problem is 5 levels. But I get why you changed it here... I just don't want anyone to forget to change it back